### PR TITLE
Add 1.26 RT Shadows

### PR DESCRIPTION
**Pull request by leonardpahlke**
Signed-off-by: leonardpahlke <leonard.pahlke@googlemail.com>

Add 1.26 RT Shadows

cc @kubernetes/release-team-leads @palnabarun

### CODE REVIEW

File: config/kubernetes/sig-release/teams.yaml [hunk 1]
```diff
@@ -281,27 +281,52 @@ teams:
...
         - jeremyrickard # subproject owner / Technical Lead
         - justaugustus # subproject owner / Chair
         - katcosgrove # 1.26 RT Lead Shadow
+        - katmutua # 1.26 Docs Shadow
         - krol3 # 1.26 Docs Lead
+        - laxmikantbpandhare # 1.26 Release Notes Shadow
         - leonardpahlke # 1.26 RT Lead
+        - marosset # 1.26 Enhancements Shadow
+        - mehabhalodiya # 1.26 CI Signal Shadow
+        - mickeyboxell # 1.26 Comms Shadow
```
Comment (reylejano) [inline config/kubernetes/sig-release/teams.yaml:310]:
```suggestion
        - mickeyboxell # 1.26 Docs Shadow
```

File: config/kubernetes/sig-release/teams.yaml [hunk 2]
```diff
@@ -281,27 +281,52 @@ teams:
...
         - palnabarun # subproject owner (1.21 RT Lead)
         members:
+        - AnaMMedina21 # 1.26 Release Notes Shadow
+        - Atharva-Shinde # 1.26 Enhancements Shadow
+        - cailynse # 1.26 Bug Triage Shadow
+        - cathchu # 1.26 Docs Shadow
         - cici37 # subproject owner (1.25 RT Lead)
         - cpanato # subproject owner / Technical Lead
+        - bharitwal # 1.26 CI Signal Shadow
+        - bradmccoydev # 1.26 Comms Lead
```
Comment (fsmunoz) [inline config/kubernetes/sig-release/teams.yaml:291]: Should be "Comms Shadow"

File: config/kubernetes/sig-release/teams.yaml [hunk 3]
```diff
@@ -281,27 +281,52 @@ teams:
...
+        - davidmirror-ops # 1.26 Comms Lead
```
Comment (fsmunoz) [inline config/kubernetes/sig-release/teams.yaml:292]: Should be "Comms Shadow"

File: config/kubernetes/sig-release/teams.yaml [hunk 4]
```diff
@@ -281,27 +281,52 @@ teams:
...
+        - Debanitrkl # 1.26 Comms Lead
```
Comment (fsmunoz) [inline config/kubernetes/sig-release/teams.yaml:293]: Should be "Comms Shadow"

**Review by fsmunoz — CHANGES_REQUESTED** [review]

**Comment by palnabarun** [comment]
We have 3 shadows pending to be added to the org.
```
    config_test.go:272: The following members of team release-team are not Kubernetes org members: heshamaboelmagd, neoaggelos, subhasmitasw
    config_test.go:272: The team release-team in org Kubernetes has an unsorted list of members 
```

**Comment by palnabarun** [comment]
Once https://github.com/kubernetes/org/pull/3708 merges, the remaining members would need to accept the org invitation request. And then this PR should be 🟢.

File: config/kubernetes/sig-release/teams.yaml [hunk 5]
```diff
@@ -281,27 +281,52 @@ teams:
...
         - palnabarun # subproject owner (1.21 RT Lead)
         members:
+        - AnaMMedina21 # 1.26 Release Notes Shadow
+        - Atharva-Shinde # 1.26 Enhancements Shadow
+        - cailynse # 1.26 Bug Triage Shadow
+        - cathchu # 1.26 Docs Shadow
         - cici37 # subproject owner (1.25 RT Lead)
         - cpanato # subproject owner / Technical Lead
+        - bharitwal # 1.26 CI Signal Shadow
+        - bradmccoydev # 1.26 Comms Shadow
```
Comment (reylejano) [inline config/kubernetes/sig-release/teams.yaml:291]: pull-org-test-all test is failing due to these entries are not sorted
please move the members starting with the letter b before the members starting with the letter c
```suggestion
        - bharitwal # 1.26 CI Signal Shadow
        - bradmccoydev # 1.26 Comms Shadow
```

**Comment by leonardpahlke** [comment]
Thanks, @palnabarun @reylejano and @fsmunoz for your reviews :) Now it looks like the PR is finally ready to get merged

**Review by palnabarun — APPROVED** [review]
/lgtm
/approve

**Pull request merged**